src/applications/travel-pay/redux/actions.js

```diff
@@ -19,11 +19,14 @@ const fetchTravelClaimsFailure = error => ({
   error,
 });
 
+/* istanbul ignore next */
 const fetchUnauthPingStart = () => ({ type: FETCH_UNAUTH_PING_STARTED });
+/* istanbul ignore next */
 const fetchUnauthPingSuccess = data => ({
   type: FETCH_UNAUTH_PING_SUCCESS,
   payload: data,
 });
+/* istanbul ignore next */
```

Comment:
Unauth ping got removed from the UI for now, so this was a false negative for poor test coveragte. We should either plan to reincorporate the ping call or remove the logic around it

Reply:
I think we should remove unauth ping code - we don't need it anymore!

Reply:
Yeah agreed, let's remove it in this PR if possible